assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -39,60 +90,118 @@ export default class AiLlmsListEditor extends Component {
+                <h2 class="admin-page-subheader__title">
+                  {{i18n "discourse_ai.llms.configured.title"}}
+                </h2>
+              </div>
+            </div>
+            <table>
+              <thead>
+                <tr>
+                  <th>{{i18n "discourse_ai.llms.display_name"}}</th>
+                  <th>{{i18n "discourse_ai.llms.provider"}}</th>
+                  <th>{{i18n "discourse_ai.llms.enabled_chat_bot"}}</th>
+                  <th></th>
                 </tr>
-              {{/each}}
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                {{#each @llms as |llm|}}
+                  <tr data-persona-id={{llm.id}} class="ai-llm-list__row">
+                    <td class="column-name">
+                      <h3>{{llm.display_name}}</h3>
+                      <p>
+                        {{i18n
+                          (concat
+                            "discourse_ai.llms.model_description."
+                            llm.provider
+                            "-"
+                            llm.name
+                          )
+                        }}
+                      </p>
+                    </td>
+                    <td>
+                      {{i18n
+                        (concat "discourse_ai.llms.providers." llm.provider)
+                      }}
+                    </td>
+                    <td>
+                      <DToggleSwitch
+                        @state={{llm.enabled_chat_bot}}
+                        {{on "click" (fn this.toggleEnabledChatBot llm)}}
+                      />
+                    </td>
+                    <td class="column-edit">
+                      <LinkTo
+                        @route="adminPlugins.show.discourse-ai-llms.show"
+                        class="btn btn-default"
+                        @model={{or llm.id llm.llm.id}}
+                      >
+                        {{icon "wrench"}}
+                        <div class="d-button-label">
+                          {{i18n "discourse_ai.llms.edit"}}
+                        </div>
+                      </LinkTo>
+                    </td>
+                  </tr>
+                {{/each}}
+              </tbody>
+            </table>
+          </section>
         {{/if}}
+        <section class="ai-llms-list-editor__templates">
+          <div class="admin-page-subheader">
+            <div class="admin-page-subheader__title-row">
+              <h2 class="admin-page-subheader__title">
+                {{#if this.hasLlmElements}}
+                  {{i18n "discourse_ai.llms.preconfigured.title"}}
+                {{else}}
+                  {{i18n "discourse_ai.llms.preconfigured.title_no_llms"}}
+                {{/if}}
+              </h2>
+            </div>
+          </div>
+          <div class="ai-llms-list-editor__templates-list">
+            {{#each this.preConfiguredLlms as |llm|}}
```

Comment:
Would really prefer these "section landing" tiles are reusable components, this is why I opened this draft PR https://github.com/discourse/discourse/pull/28477/files . At the least could you please extract this to a component in this PR then we can replace with a core version later?

Reply:
lets do this in a followup, really want to land this. I addressed the rest of the feedback.